OptionSelector/OSCategoriesTableViewController.m

```diff
@@ -0,0 +1,125 @@
+//
+//  OSCategoriesTableViewController.m
+//  OptionSelector
+//
+//  Created by Daniel Distant on 8/11/15.
+//  Copyright (c) 2015 Mike Kavouras. All rights reserved.
+//
+
+#import "OSCategoriesTableViewController.h"
+#import "CQCategory.h"
+#import "OSStuffTableViewController.h"
+
+@interface OSCategoriesTableViewController () <UITableViewDelegate>
+
+@property (nonatomic) NSArray *CQCategories;
+
```

Comment:
Style: Try to keep your vertical spacing consistent (there should be one empty line between the last `@property` and `@end`).